bubblewrap.c

```diff
@@ -2162,7 +2168,7 @@ main (int    argc,
 
   __debug__ (("forking for child\n"));
 
-  if (opt_unshare_pid || lock_files != NULL || opt_sync_fd != -1)
+  if (!no_reaper && (opt_unshare_pid || lock_files != NULL || opt_sync_fd != -1))
```

Comment:
hm.. doesn't opt_die_with_parent still work without pid1?  The `handle_die_with_parent()` just before the execvp will still be valid no?

I've done a quick test and the process in the sandbox exits when bwrap is killed.